shared/react-native/android/app/build.gradle

```diff
@@ -88,13 +88,14 @@ android {
     dexOptions {
         jumboMode true
     }
-    compileSdkVersion 27
-    buildToolsVersion "27.0.3"
+
```

Comment:
new camera lib needs updated gradle. We need to update this for RN 0.56 anyways so might as well